### DISCUSSION TITLE
[A-L] (2023/24) Foglio 8 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#318] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-17 184653](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/35d44f30-77de-4e42-95bb-b3e77060c1c2)

**Comment by daromacarlo** [comment]
Prima permutazione)
![Immagine 2023-11-23 192303](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/accc2da6-12d8-487b-be37-845dcc5e1fe7)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/c4f1e89d-8419-43b5-9d69-1b6861eab1e2)

**Comment by daromacarlo** [comment]
Seconda permutazione)
![Immagine 2024-01-06 180828](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/c60fcdab-53ca-4567-968e-5fd8fc36de47)
